2015-10-22 18:49:27 Error: [Cake\View\Exception\MissingTemplateException] Template file "Articles\view.ctp" is missing.
Exception Attributes: array (
  'file' => 'Articles\\view.ctp',
)
Request URL: /xamppA/articles/view/1
Stack Trace:
#0 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\View\View.php(575): Cake\View\View->_getViewFileName(NULL)
#1 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Controller\Controller.php(611): Cake\View\View->render(NULL, NULL)
#2 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(120): Cake\Controller\Controller->render()
#3 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\ArticlesController))
#4 C:\xampp\htdocs\xamppA\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#5 {main}


2015-10-22 18:51:22 Error: [Cake\View\Exception\MissingTemplateException] Template file "Articles\view.ctp" is missing.
Exception Attributes: array (
  'file' => 'Articles\\view.ctp',
)
Request URL: /xamppA/articles/view/1
Stack Trace:
#0 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\View\View.php(575): Cake\View\View->_getViewFileName(NULL)
#1 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Controller\Controller.php(611): Cake\View\View->render(NULL, NULL)
#2 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(120): Cake\Controller\Controller->render()
#3 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\ArticlesController))
#4 C:\xampp\htdocs\xamppA\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#5 {main}


2015-10-22 18:52:41 Error: [Cake\View\Exception\MissingTemplateException] Template file "Articles\view.ctp" is missing.
Exception Attributes: array (
  'file' => 'Articles\\view.ctp',
)
Request URL: /xamppA/articles/view/1
Stack Trace:
#0 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\View\View.php(575): Cake\View\View->_getViewFileName(NULL)
#1 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Controller\Controller.php(611): Cake\View\View->render(NULL, NULL)
#2 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(120): Cake\Controller\Controller->render()
#3 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\ArticlesController))
#4 C:\xampp\htdocs\xamppA\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#5 {main}


2015-10-22 18:55:20 Error: Fatal Error (4): syntax error, unexpected end of file, expecting '`' in [C:\xampp\htdocs\xamppA\src\Controller\ArticlesController.php, line 110]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-10-22 18:55:20 Error: [Cake\Error\FatalErrorException] syntax error, unexpected end of file, expecting '`'
Request URL: /xamppA/Articles/index
Stack Trace:
#0 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 110)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-10-22 19:02:37 Error: [Cake\Datasource\Exception\RecordNotFoundException] Record not found in table "articles"
Request URL: /xamppA/articles/view/9
Stack Trace:
#0 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\ORM\Table.php(1187): Cake\ORM\Query->firstOrFail()
#1 C:\xampp\htdocs\xamppA\src\Controller\ArticlesController.php(43): Cake\ORM\Table->get('9', Array)
#2 [internal function]: App\Controller\ArticlesController->view('9')
#3 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Controller\Controller.php(429): call_user_func_array(Array, Array)
#4 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(114): Cake\Controller\Controller->invokeAction()
#5 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\ArticlesController))
#6 C:\xampp\htdocs\xamppA\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#7 {main}


2015-10-29 17:17:46 Error: [PDOException] SQLSTATE[HY000] [1049] Unknown database 'sis'
Request URL: /xamppA/
Stack Trace:
#0 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Database\Driver\PDODriverTrait.php(48): PDO->__construct('mysql:host=loca...', 'root', '', Array)
#1 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Database\Driver\Mysql.php(90): Cake\Database\Driver\Mysql->_connect('mysql:host=loca...', Array)
#2 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Database\Schema\BaseSchema.php(45): Cake\Database\Driver\Mysql->connect()
#3 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Database\Dialect\MysqlDialectTrait.php(63): Cake\Database\Schema\BaseSchema->__construct(Object(Cake\Database\Driver\Mysql))
#4 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Database\Schema\Collection.php(52): Cake\Database\Driver\Mysql->schemaDialect()
#5 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Database\Schema\CachedCollection.php(43): Cake\Database\Schema\Collection->__construct(Object(Cake\Database\Connection))
#6 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Database\Connection.php(315): Cake\Database\Schema\CachedCollection->__construct(Object(Cake\Database\Connection), true)
#7 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\ORM\Table.php(422): Cake\Database\Connection->schemaCollection()
#8 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\ORM\Query.php(164): Cake\ORM\Table->schema()
#9 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\ORM\Query.php(122): Cake\ORM\Query->addDefaultTypes(Object(App\Model\Table\ArticlesTable))
#10 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\ORM\Table.php(1228): Cake\ORM\Query->__construct(Object(Cake\Database\Connection), Object(App\Model\Table\ArticlesTable))
#11 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\ORM\Table.php(931): Cake\ORM\Table->query()
#12 C:\xampp\htdocs\xamppA\src\Controller\ArticlesController.php(27): Cake\ORM\Table->find('all')
#13 [internal function]: App\Controller\ArticlesController->index()
#14 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Controller\Controller.php(429): call_user_func_array(Array, Array)
#15 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(114): Cake\Controller\Controller->invokeAction()
#16 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\ArticlesController))
#17 C:\xampp\htdocs\xamppA\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#18 {main}


2015-10-29 17:26:28 Error: Fatal Error (1): Call to a member function format() on null in [C:\xampp\htdocs\xamppA\src\Template\Articles\index.ctp, line 19]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-10-29 17:26:28 Error: [Cake\Error\FatalErrorException] Call to a member function format() on null
Request URL: /xamppA/
Stack Trace:
#0 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 19)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-10-29 17:32:46 Error: Fatal Error (1): Call to a member function format() on null in [C:\xampp\htdocs\xamppA\src\Template\Articles\index.ctp, line 19]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-10-29 17:32:46 Error: [Cake\Error\FatalErrorException] Call to a member function format() on null
Request URL: /xamppA/
Stack Trace:
#0 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 19)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-10-29 18:11:53 Error: Fatal Error (1): Call to a member function format() on null in [C:\xampp\htdocs\xamppA\src\Template\Articles\index.ctp, line 19]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 191
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 90
[main] - [internal], line ??



2015-10-29 18:11:53 Error: [Cake\Error\FatalErrorException] Call to a member function format() on null
Request URL: /xamppA/
Stack Trace:
#0 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(90): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 19)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-10-29 18:28:30 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Add could not be found.
Exception Attributes: array (
  'class' => 'Add',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /xamppA/add/user
Stack Trace:
#0 C:\xampp\htdocs\xamppA\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-10-29 18:36:14 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Add could not be found.
Exception Attributes: array (
  'class' => 'Add',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /xamppA/add/user
Stack Trace:
#0 C:\xampp\htdocs\xamppA\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-10-29 18:36:26 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Add could not be found.
Exception Attributes: array (
  'class' => 'Add',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /xamppA/add/user
Stack Trace:
#0 C:\xampp\htdocs\xamppA\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-10-29 18:55:21 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Users could not be found.
Exception Attributes: array (
  'class' => 'Users',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /xamppA/users/login
Stack Trace:
#0 C:\xampp\htdocs\xamppA\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-10-29 18:55:48 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Users could not be found.
Exception Attributes: array (
  'class' => 'Users',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /xamppA/users/login
Stack Trace:
#0 C:\xampp\htdocs\xamppA\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-10-29 19:13:05 Error: [PDOException] SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '0' for key 'PRIMARY'
Request URL: /xamppA/users/add
Stack Trace:
#0 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Database\Statement\MysqlStatement.php(36): PDOStatement->execute(NULL)
#1 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Database\Statement\StatementDecorator.php(171): Cake\Database\Statement\MysqlStatement->execute(NULL)
#2 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Database\Log\LoggingStatement.php(56): Cake\Database\Statement\StatementDecorator->execute(NULL)
#3 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Database\Connection.php(270): Cake\Database\Log\LoggingStatement->execute()
#4 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Database\Query.php(174): Cake\Database\Connection->run(Object(Cake\ORM\Query))
#5 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\ORM\Table.php(1530): Cake\Database\Query->execute()
#6 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\ORM\Table.php(1446): Cake\ORM\Table->_insert(Object(Cake\ORM\Entity), Array)
#7 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\ORM\Table.php(1377): Cake\ORM\Table->_processSave(Object(Cake\ORM\Entity), Object(ArrayObject))
#8 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Database\Connection.php(546): Cake\ORM\Table->Cake\ORM\{closure}(Object(Cake\Database\Connection))
#9 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\ORM\Table.php(1378): Cake\Database\Connection->transactional(Object(Closure))
#10 C:\xampp\htdocs\xamppA\src\Controller\UsersController.php(47): Cake\ORM\Table->save(Object(Cake\ORM\Entity))
#11 [internal function]: App\Controller\UsersController->add()
#12 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Controller\Controller.php(429): call_user_func_array(Array, Array)
#13 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(114): Cake\Controller\Controller->invokeAction()
#14 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\UsersController))
#15 C:\xampp\htdocs\xamppA\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#16 {main}


2015-10-29 19:13:22 Error: [PDOException] SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '0' for key 'PRIMARY'
Request URL: /xamppA/users/add
Stack Trace:
#0 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Database\Statement\MysqlStatement.php(36): PDOStatement->execute(NULL)
#1 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Database\Statement\StatementDecorator.php(171): Cake\Database\Statement\MysqlStatement->execute(NULL)
#2 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Database\Log\LoggingStatement.php(56): Cake\Database\Statement\StatementDecorator->execute(NULL)
#3 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Database\Connection.php(270): Cake\Database\Log\LoggingStatement->execute()
#4 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Database\Query.php(174): Cake\Database\Connection->run(Object(Cake\ORM\Query))
#5 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\ORM\Table.php(1530): Cake\Database\Query->execute()
#6 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\ORM\Table.php(1446): Cake\ORM\Table->_insert(Object(Cake\ORM\Entity), Array)
#7 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\ORM\Table.php(1377): Cake\ORM\Table->_processSave(Object(Cake\ORM\Entity), Object(ArrayObject))
#8 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Database\Connection.php(546): Cake\ORM\Table->Cake\ORM\{closure}(Object(Cake\Database\Connection))
#9 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\ORM\Table.php(1378): Cake\Database\Connection->transactional(Object(Closure))
#10 C:\xampp\htdocs\xamppA\src\Controller\UsersController.php(47): Cake\ORM\Table->save(Object(Cake\ORM\Entity))
#11 [internal function]: App\Controller\UsersController->add()
#12 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Controller\Controller.php(429): call_user_func_array(Array, Array)
#13 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(114): Cake\Controller\Controller->invokeAction()
#14 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\UsersController))
#15 C:\xampp\htdocs\xamppA\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#16 {main}


2015-10-29 19:19:57 Error: [PDOException] SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '0' for key 'PRIMARY'
Request URL: /xamppA/users/add
Stack Trace:
#0 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Database\Statement\MysqlStatement.php(36): PDOStatement->execute(NULL)
#1 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Database\Statement\StatementDecorator.php(171): Cake\Database\Statement\MysqlStatement->execute(NULL)
#2 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Database\Log\LoggingStatement.php(56): Cake\Database\Statement\StatementDecorator->execute(NULL)
#3 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Database\Connection.php(270): Cake\Database\Log\LoggingStatement->execute()
#4 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Database\Query.php(174): Cake\Database\Connection->run(Object(Cake\ORM\Query))
#5 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\ORM\Table.php(1530): Cake\Database\Query->execute()
#6 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\ORM\Table.php(1446): Cake\ORM\Table->_insert(Object(Cake\ORM\Entity), Array)
#7 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\ORM\Table.php(1377): Cake\ORM\Table->_processSave(Object(Cake\ORM\Entity), Object(ArrayObject))
#8 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Database\Connection.php(546): Cake\ORM\Table->Cake\ORM\{closure}(Object(Cake\Database\Connection))
#9 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\ORM\Table.php(1378): Cake\Database\Connection->transactional(Object(Closure))
#10 C:\xampp\htdocs\xamppA\src\Controller\UsersController.php(47): Cake\ORM\Table->save(Object(Cake\ORM\Entity))
#11 [internal function]: App\Controller\UsersController->add()
#12 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Controller\Controller.php(429): call_user_func_array(Array, Array)
#13 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(114): Cake\Controller\Controller->invokeAction()
#14 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\UsersController))
#15 C:\xampp\htdocs\xamppA\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#16 {main}


2015-12-19 19:57:35 Error: [PDOException] SQLSTATE[HY000] [1049] Unknown database 'sis'
Request URL: /xamppA/
Stack Trace:
#0 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Database\Driver\PDODriverTrait.php(48): PDO->__construct('mysql:host=loca...', 'root', '', Array)
#1 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Database\Driver\Mysql.php(90): Cake\Database\Driver\Mysql->_connect('mysql:host=loca...', Array)
#2 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Database\Schema\BaseSchema.php(45): Cake\Database\Driver\Mysql->connect()
#3 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Database\Dialect\MysqlDialectTrait.php(63): Cake\Database\Schema\BaseSchema->__construct(Object(Cake\Database\Driver\Mysql))
#4 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Database\Schema\Collection.php(52): Cake\Database\Driver\Mysql->schemaDialect()
#5 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Database\Schema\CachedCollection.php(43): Cake\Database\Schema\Collection->__construct(Object(Cake\Database\Connection))
#6 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Database\Connection.php(315): Cake\Database\Schema\CachedCollection->__construct(Object(Cake\Database\Connection), true)
#7 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\ORM\Table.php(422): Cake\Database\Connection->schemaCollection()
#8 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\ORM\Query.php(164): Cake\ORM\Table->schema()
#9 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\ORM\Query.php(122): Cake\ORM\Query->addDefaultTypes(Object(App\Model\Table\ArticlesTable))
#10 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\ORM\Table.php(1228): Cake\ORM\Query->__construct(Object(Cake\Database\Connection), Object(App\Model\Table\ArticlesTable))
#11 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\ORM\Table.php(931): Cake\ORM\Table->query()
#12 C:\xampp\htdocs\xamppA\src\Controller\ArticlesController.php(27): Cake\ORM\Table->find('all')
#13 [internal function]: App\Controller\ArticlesController->index()
#14 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Controller\Controller.php(429): call_user_func_array(Array, Array)
#15 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(114): Cake\Controller\Controller->invokeAction()
#16 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\ArticlesController))
#17 C:\xampp\htdocs\xamppA\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#18 {main}


2015-12-19 20:44:25 Error: [Cake\View\Exception\MissingTemplateException] Template file "Users\index.ctp" is missing.
Exception Attributes: array (
  'file' => 'Users\\index.ctp',
)
Request URL: /xamppA/users/
Stack Trace:
#0 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\View\View.php(575): Cake\View\View->_getViewFileName(NULL)
#1 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Controller\Controller.php(611): Cake\View\View->render(NULL, NULL)
#2 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(120): Cake\Controller\Controller->render()
#3 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\UsersController))
#4 C:\xampp\htdocs\xamppA\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#5 {main}


2015-12-19 20:44:31 Error: [Cake\View\Exception\MissingTemplateException] Template file "Users\index.ctp" is missing.
Exception Attributes: array (
  'file' => 'Users\\index.ctp',
)
Request URL: /xamppA/users
Stack Trace:
#0 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\View\View.php(575): Cake\View\View->_getViewFileName(NULL)
#1 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Controller\Controller.php(611): Cake\View\View->render(NULL, NULL)
#2 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(120): Cake\Controller\Controller->render()
#3 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\UsersController))
#4 C:\xampp\htdocs\xamppA\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#5 {main}


2015-12-19 20:48:07 Error: [Cake\View\Exception\MissingTemplateException] Template file "Users\index.ctp" is missing.
Exception Attributes: array (
  'file' => 'Users\\index.ctp',
)
Request URL: /xamppA/users
Stack Trace:
#0 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\View\View.php(575): Cake\View\View->_getViewFileName(NULL)
#1 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Controller\Controller.php(611): Cake\View\View->render(NULL, NULL)
#2 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(120): Cake\Controller\Controller->render()
#3 C:\xampp\htdocs\xamppA\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\UsersController))
#4 C:\xampp\htdocs\xamppA\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#5 {main}


